### PR TITLE
corrections

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: DayrSuryan/DSEthiop7.xml [hunk 1]
```diff
@@ -361,7 +361,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <item xml:id="a1">
                            <locus target="#iv"/>
                            <desc type="OwnershipNote">Ownership note stating that the manuscript belongs to
-                              <persName ref="PRS12557Barha"><roleName type="title"><hi rendition="simple:italic">Ḥalaqā</hi></roleName> Barha</persName>.
+                              <persName ref="PRS12557Barha"><roleName type="title"><hi rendition="simple:italic">Ḫalaqā</hi></roleName> Barha</persName>.
```
Comment (DenisNosnitsin1970) [inline DayrSuryan/DSEthiop7.xml:364]: If we transliterate, the name of the brave Ḫalaqā should be spelled Barhe.

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop7.xml:364]: I suggest that we remove (!) after his name. በርሄ is not a deviation, it is the old Tigrigna orthography

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: DayrSuryan/DSEthiop6.xml [hunk 2]
```diff
@@ -198,12 +198,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         <item xml:id="a5">
                            <locus target="#110v"/>
-                           <desc type="Supplication">Short supplication in Tigrinya. The number 43 in the beginning
+                           <desc type="Supplication">Short supplication in Tigrinya. The number 43 at the beginning
                              may refer to the year
                              1943 <hi rendition="simple:smallcaps">EC</hi> (= 1950/1951 <hi rendition="simple:smallcaps">CE</hi>).</desc>
                            <q xml:lang="ti"><add place="above">፵፫</add>
                               ሃይማኖት፡ እስክ<add place="above">ን</add>ድርያ፡ መንግሥቲ፡ ኢ<add place="above">ት</add>ዮጵያ፡
-                           ይዕቀበልና፡ አሜን፡ ወአምየን።</q>
+                           ይዕቀበልና፡ አሜን፡ ወአም<sic>የን</sic>።</q>
```
[the remark this answers is not in the document]
Comment (thea-m) [reply DayrSuryan/DSEthiop6.xml:206]: It is like this in the manuscript

File: DayrSuryan/DSEthiop7.xml (hunk 1 above)
Comment (thea-m) [reply DayrSuryan/DSEthiop7.xml:364]: but it is በሬሄ here

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop7.xml]: I see, it interfered with PRS12557Barh**a**

Comment (thea-m) [reply DayrSuryan/DSEthiop7.xml]: I think the ID doesn't matter, but the sic refers to ሬ

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**